cloud/integrations.md

```diff
@@ -60,8 +72,8 @@ Create a data exporter to send telemetry data to your preferred provider.
 1.  Enter your AWS credentials. To get your AWS keys, see [Getting access keys
     for AWS CloudWatch][aws-access-keys].
 1.  Select an AWS Region for your CloudWatch instance.
-1.  **(optional)** Define an IAM role to upload segments to a
-    different account.
+1.  **OPTIONAL** Define an Identity and Access Management (IAM) role to upload
```

Comment:
```suggestion
1.  **OPTIONAL** Define an identity and access management (IAM) role to upload
```

Only proper nouns take initcaps 😉

Reply:
I think it's the name of an actual thing in AWS and they style it with caps: https://aws.amazon.com/iam/ :)

Reply:
ugh. OK 🤦‍♀️